.circleci/config.yml

```diff
@@ -32,24 +34,29 @@ jobs:
       - run:
           name:  "Test: Execute Python OWASP ZAP Vulnerability Scan"
           command: "cd tdrs-backend; ./zap-scanner.sh"
+      - store_artifacts:
+          path: tdrs-backend/reports/owasp_report.html
   test-frontend:
     working_directory: ~/tdp-apps
-    docker:
-      - image: cypress/base:10
-        user : root
+    machine:
```

Comment:
We needed to move to the [machine executor](https://circleci.com/docs/2.0/executor-types/#using-machine) in order to be able to use a mounted volume to copy the generated OWASP report to the CI host, where it can be stored by the `store_artifacts` command.

[This is not supported on the docker executor we were previously using.](https://support.circleci.com/hc/en-us/articles/360007324514-How-can-I-use-Docker-volume-mounting-on-CircleCI-)